<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeWidget</class>
 <widget class="QWidget" name="AttributeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_55">
       <property name="title">
        <string>Attribute Type</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioWeapon">
          <property name="text">
           <string>Weapon</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioMagic">
          <property name="text">
           <string>Magic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_56">
       <property name="title">
        <string>Damage Multiplier</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="pixmap">
           <pixmap>:/ranks/rank0</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinA">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="pixmap">
           <pixmap>:/ranks/rank1</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinB">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="pixmap">
           <pixmap>:/ranks/rank2</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinC">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="pixmap">
           <pixmap>:/ranks/rank3</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinD">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="pixmap">
           <pixmap>:/ranks/rank4</pixmap>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinE">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <number>-99999</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>367</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
